src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+// import { type } from 'os';
+
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(({
+        title,
+        description,
```

Comment:
`description` is a not required field, it might be undefined, you should set some default value for it